### PR TITLE
[SFSRAP-37] Remove JDK-related code and resources

### PR DESCRIPTION
**Pull request by duncanpMS**
Updated the readme: the JDK is no longer a pre-requisite

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,13 +24,9 @@ The tooling is at an early stage and only works in limited scenarios. Also, the
    - the analyzer must be available as a NuGet package
    - the analyzer must use Roslyn v1.0 or v1.1
    - only C# rules are supported
-   - the NuGet package must not require user acceptance of the license
 
 These limitations will be addressed at some point in the future.
 
-#### Pre-requisities
-The SDK uses the Java compiler and jar packaging tool so you will need to run the SDK exe on a machine with version 7 of the [Java SE development kit](http://www.oracle.com/technetwork/java/javase/overview/index.html). The SonarQube site [lists](http://docs.sonarqube.org/display/SONAR/Requirements) the supported Java implementations. However, for the plugin to work correctly it should target the correct Java version, currently version 7. See [SFSRAP-35](https://jira.sonarsource.com/browse/SFSRAP-35) for more information. We'll improve how the SDK handles Java version mismatches, but for now you will need to make sure you target the correct version.
-
```
Comment (duncanpMS) [inline README.md]: Updated the readme to remove the reference to the JDK.

Comment (duncanpMS) [inline README.md]: There are no changes to the product code at all; I've just deleted the now-redundant classes, tests and resource strings.

**Comment by dbolkensteyn** [comment]
Nice! LGTM

**Comment by duncanpMS** [comment]
There were 4121 lines of code before this commit; it'll be interesting to see how many there are afterwards...

**Pull request merged**